Source/Application/Do-Svyazi.User.Application/CQRS/Authenticate/Handlers/AuthenticateCommandHandler.cs

```diff
@@ -1,72 +1,71 @@
 using Do_Svyazi.User.Application.CQRS.Authenticate.Commands;
 using Do_Svyazi.User.Application.CQRS.Handlers;
+using Do_Svyazi.User.Application.DbContexts;
 using Do_Svyazi.User.Domain.Authenticate;
 using Do_Svyazi.User.Domain.Exceptions;
 using MediatR;
 using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
 
 namespace Do_Svyazi.User.Application.CQRS.Authenticate.Handlers;
 
-public class AuthenticateCommandHandler
```

Comment:
Maybe it'll make sense to create messengerUsers in register commands...

@ronimizy